config/action.d/pf.conf

```diff
@@ -75,14 +75,13 @@ protocol = tcp
 # Option:  port
 # Notes.:  specifies port to monitor
 # Values:  [ NUM | STRING ]  Default:
-#
-#port = telnet
+port = ssh
```

Comment:
>  > -#port = telnet
>  > +port = ssh
>    Ahh, I see the port defined here will be not used at all, because uses
>    always the port defined in jail.conf (or .local)

yes!  This is just a "sensible default".  And in general agree, as with
you other comment, ideally we fail than operate on some "random" port.
But if we are to make such decision to change behavior, we better apply
it across all actions, not just one at a time.  That is why this change
was to stay consistent with other actions/current "ideology".